shell/platform/darwin/common/framework/Headers/FlutterMacros.h

```diff
@@ -6,12 +6,17 @@
 #define FLUTTER_FLUTTERMACROS_H_
 
 #if defined(FLUTTER_FRAMEWORK)
-
+// Overrides define in embedder API
+#ifdef FLUTTER_EXPORT
+#undef FLUTTER_EXPORT
+#endif
```

Comment:
This seems off to me, but maybe I'm missing something.

Should we just be renaming the macro in this file?

/cc @stuartmorgan @chinmaygarde

Reply:
for reference, If i don't fix this conflict, i need to import embedder.h before importing /Headers/FlutterEngine.h in some .mm file

Reply:
filed https://github.com/flutter/flutter/issues/74679
with pr https://github.com/flutter/engine/pull/23933

Reply:
Is this still necessary after #23933?

Reply:
Was looking at an outdated diff. Marking resolved.